### ISSUE TITLE
About testing set of NPC170 Dataset

### ISSUE THREAD

**Issue by kunaljangid01** [ycwu1997/D-Persona#6]
Hi,
I got the training and validation sets of the NPC170 dataset from the "**MMIS2024 challenge**". 
But the test set is not included. Could you please provide the test set for the NPC170 dataset?

I have registered for the **MMIS2024 challenge** using the email **kunal24@iiserb.ac.in**.

Please provide the testing set or email me.

Thank you for your time and consideration.
